docker/duckdb_server.py

```
@@ -0,0 +1,62 @@
+
+
+@app.get("/health")
+async def health_check() -> dict[str, str]:
+    """Check if the database is accessible."""
+    try:
+        result = conn.execute("SELECT 1")
+        result.fetchone()
+        return {"status": "healthy"}
+    except Exception as e:
+        logger.error(f"Health check failed: {str(e)}")
+        raise HTTPException(status_code=503, detail=str(e))
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)